flipperdevices/flipperzero-firmware
# FuriHal, About, Desktop: show provisioned region

# What's new

- FuriHal, About, Desktop: show provisioned region

# Verification 

- Compile and upload
- Check debug info screen
- Check about

# Checklist (For Reviewer)
